examples/transformerconfigs/README.md

```diff
@@ -606,3 +606,63 @@ metadata:
     foo.k8s.io/bar: baz
 ```
 Kustomize supports escaping special characters in path, e.g `metadata/annotations/foo.k8s.io\/bar`
+
+## Namespace
+
+The namespace transformer is used to update namespace references in your manifests when you set the `namespace` declaration in your `kustomization.yaml`.
+
+### Example
+
+kustomization.yaml
+
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: example
+
+resources:
+- resources.yaml
+
+configurations:
+  - namespace.yaml
```

Comment:
Please remove spaces from the extra indent on this list.
I think our style is no extra indent for lists.